### ISSUE TITLE
Feature request: page down/up should go to the next/previous section

### ISSUE THREAD

**Issue by rellfy** [heyman/heynote#72]
I think this would make much more sense for the app than the default page up/down behaviour.

**Comment by ImKaashif** [comment]
do you mean something similar to `Ctrl` + `Down` and `Ctrl` + `Up` ?

**Comment by rellfy** [comment]
> do you mean something similar to `Ctrl` + `Down` and `Ctrl` + `Up` ?

@ImKaashif Yes. Although I would expect it to go to the beginning of the next block, not the end.

**Comment by heyman** [comment]
I don't want to override the default of Page up/down. However, when proper keybindings support is implemented, this will be possible to configure yourself.

**Comment by llagerlof** [comment]
> > do you mean something similar to `Ctrl` + `Down` and `Ctrl` + `Up` ?
> 
> @ImKaashif Yes. Although I would expect it to go to the beginning of the next block, not the end.

About this, I am having a strange behavior using `Ctrl` + `Down` and `Ctrl` + `Up` . The little doc at the top says:
```
Ctrl + Down            Goto next block
Ctrl + Up              Goto previous block
```

But when I press `Ctrl` + `Down` the cursor goes to the end of the current block, not to the next block below the current one. 

When I press `Ctrl` + `Up` the cursor goes to the top of the current block, not to the previous block above the current one.

**Comment by heyman** [comment]
That's the expected behaviour. If the cursor is in the middle of a block, the first time you press `Ctrl + Up/Down` the cursor will go to the start/end of the block. Subsequent `Ctrl + up/down` will jump to the previous/next block.

**Comment by heyman** [comment]
Custom key bindings have been implemented.
